siddpubs-conf.bib

```diff
@@ -64,12 +64,19 @@ @string{ldc
 @string{emnlp     = "Empirical Methods in Natural Language Processing"}
 
 % 2025 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+@inproceedings{lin2025ccdiff,
+  title={Causal Composition Diffusion Model for Closed-loop Traffic Generation}, 
+  author={Lin, H. and Huang, X. and Phan-Minh, T. and Hayden, D.S. and Zhang, H. and Zhao, D. and Srinivasa, S.S. and Wolff, E.M. and Chen, H.},
+  booktitle = cvpr,
+  year = {2025}
+}
+
 @inproceedings{nanavati2025lessons,
   title={Lessons Learned from Designing and Evaluating a Robot-assisted Feeding System for Out-of-lab Use},
   author={Nanavati, Amal and Gordon, Ethan K and Kessler Faulkner, Taylor A and Song, Yuxin (Ray) and Ko, Johnathan and Schrenk, Tyler and Nguyen, Vy and Zhu, Bernie Hao and Bolotski, Haya and Kashyap, Atharva and Kutty, Sriram and Karim, Raida and Rainbolt, Liander and Scalise, Rosario and Song, Hanjun and Qu, Ramon and Cakmak, Maya and Srinivasa, Siddhartha S},
   booktitle = hri,
   year={2025},
-  note = {{\href{Website}{https://robotfeeding.io/publications/hri25a/}}}
+  note = {{\href{Website}{https://robotfeeding.io/publications/hri25a/}, \textbf{Best Paper Award Finalist}}}
```

Comment:
Amal and I learned that the website cannot render multiple references in a note properly (we tried this earlier and had to undo it - see https://github.com/personalrobotics/pubs/commit/4a12d883cdacc22cf9c996f0e32baf4bb98b840d). Can this change be reverted to have the note point to the paper website only? The paper website makes note of the best paper award finalist.

Reply:
Adding some details: the function that converts latex to HTML for the website, defined [here](https://github.com/personalrobotics/prl-website/blob/89de50c19556a1a2e6856e2af9024092b5ab5333/personalrobotics/main/publications.py#L264), can only handles a single `textbf` and/or a single `href` at the beginning of the latex string. Addressing that would require modifying the regexes to work in the middle of the string, and applying the regex substitution multiple times (perhaps recursively) on the input string.

Reply:
How about a compromise like this - 
`note = {{\href{Website, Best Systems Paper Award Finalist}{https://robotfeeding.io/publications/hri25a/}}`

Reply:
Looks perfect to me 👍🏽

Reply:
That's also fine with me!